ASM/c/z64.h

```diff
@@ -2081,6 +2097,8 @@ typedef uint16_t (*z64_Audio_GetActiveSeqId_proc)(uint8_t seqId);
                                                       z64_DisplayTextbox_addr)
 #define z64_GiveItem            ((z64_GiveItem_proc)  z64_GiveItem_addr)
 
+void z64_Inventory_ChangeUpgrade(int16_t upgrade, int16_t value);
```

Comment:
Is there a reason you did this non-standard to what has already been done here?

Reply:
Because two reasons. First, I think it looks cleaner this way. Second, I'm not sure how to write a pointer to a double array properly (`uint16_t gUpgradeCapacities[UPG_MAX][4]`)

Reply:
Okay, I think this should be equivalent
```c

#define gUpgradeCapacities      ( (uint16_t (*)[4]) gUpgradeCapacities_addr)
```